src/services/geometry/richgeo/RichGeo.h

```diff
@@ -7,9 +7,12 @@
 #include <iostream>
 #include <fmt/format.h>
 #include <spdlog/spdlog.h>
+#include <edm4hep/SimTrackerHitData.h>
 
 namespace richgeo {
 
+  typedef decltype(edm4hep::SimTrackerHitData::cellID) CellIDType;
```

Comment:
`using` is preferred in modern C++.
```suggestion
  using CellIDType = decltype(edm4hep::SimTrackerHitData::cellID);
```
Ref. https://www.nextptr.com/tutorial/ta1193988140/how-cplusplus-using-or-aliasdeclaration-is-better-than-typedef (just a first hit on google)

Reply:
I also think `*_t` as a type alias is a more typical naming convention.

Reply:
So maybe `cellID_t` or something?

Reply:
It doesn't seem like we have any convention for alias names here in EICrecon.

Reply:
Yeah, I don't feel strongly about this. Absent formatting guidance, I'll resolve this.